### PR TITLE
Document getConfirmation JSON RPC API

### PR DESCRIPTION
**Pull request by mvines**

### CODE REVIEW

File: book/src/jsonrpc-api.md [hunk 1]
```diff
@@ -201,6 +202,25 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 {"jsonrpc":"2.0","result":268,"id":1}
 ```
 
+---
+### getConfirmation
+Returns the current cluster confirmation time in milliseconds
```
Comment (garious) [inline book/src/jsonrpc-api.md]: current? How about "Returns the mean confirmation time (in milliseconds) within the last X seconds" where X is some constant.

File: book/src/jsonrpc-api.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ Methods
 * [getLastId](#getlastid)
 * [getSignatureStatus](#getsignaturestatus)
 * [getTransactionCount](#gettransactioncount)
+* [getConfirmation](#getconfirmation)
```
Comment (CriesofCarrots) [inline book/src/jsonrpc-api.md]: How about `getConfirmationTime`? I find `getConfirmation` to be a little ambiguous.
Also, let's alphabetize this list?

Comment (garious) [reply book/src/jsonrpc-api.md]: Maybe even getMeanConfirmationTime to distinguish it from furture API that returns the confirmation time of a particular block?

Comment (mvines) [reply book/src/jsonrpc-api.md]: It returns the confirmation time of the latest block, the current API is not that smart: https://github.com/solana-labs/solana/blob/16877ecf037ae2b41cb2ceaab1fb5048a61f9d9c/src/compute_leader_confirmation_service.rs#L75

File: book/src/jsonrpc-api.md (hunk 1 above)
Comment (mvines) [reply book/src/jsonrpc-api.md]: But it doesn't actually do that

**Pull request merged**